### DISCUSSION TITLE
Mergerfs is writing to the second drive instead of the first

### DISCUSSION THREAD

**Discussion by gnuyen** [trapexit/mergerfs#971] [Q&A]
Mounting using command

/usr/bin/mergerfs -o defaults,rw,sync_read,auto_cache,use_ino,allow_other,func.getattr=newest,category.action=ff,category.create=ff /mnt/local/storage/jellyfin/TV:/mnt/remote/PlexMediaRO/TV /mnt/merged/jellyfin/TV >> /var/log/rclone/jellyfin_mount_tv.log

and newly created files are being written to the PlexMEdiaRO/TV drive instead of the /mnt/local/storage/jellyfin/TV file.  

Am I doing something wrong?   The /mnt/remote/PlexMediaRO is an rclone mount (so also fuse)

mergerfs version: 2.28.1
FUSE library version: 2.9.7-mergerfs_2.28.0
fusermount version: 2.9.9
using FUSE kernel interface version 7.29

**Comment by trapexit** [comment]
1) Please use current releases of the software
2) You have to provide *all* data relevant to your config. Everything related to the policy you use. Free space, readonly status, etc.

The policies work *exactly* as described. Just walk through the behavior and you'll get your answer. 

https://github.com/trapexit/mergerfs#filtering
